.byebug_history

```diff
@@ -0,0 +1,4 @@
+c
```

Comment:
Why are we committing .byebug history?
📌

Reply:
It was not known to me that its byebug creates a history file, will be sure to remove it next time